src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.netcore.cs

```diff
@@ -14546,11 +14624,12 @@ public static Vector512<float> Invoke(Vector512<float> x)
         internal readonly struct Log10Operator<T> : IUnaryOperator<T, T>
             where T : ILogarithmicFunctions<T>
         {
-            public static bool Vectorizable => false; // TODO: Vectorize
+            private static readonly double s_log10 = Math.Log(10d);
```

Comment:
Not `NaturalLog10` like the other cases?

Reply:
This one's not a constant. I could name it s_naturalLog10 instead.

Reply:
Why isn't/can't it be a constant? The nearest representable result to the infinitely precise answer is the same as `NaturalLog10` used in another function further up (`2.3025850929940456840179914546843642076011014886287729760333279009...` whose shortest roundtrippable value as a double is `2.302585092994046`).

Reply:
Because... no reason. I'd previously had this as computing a value per T, at which point it had to be a static readonly. I switched it to double and didn't re-evaluate.